.github/config.yml

```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery/master/.github/welcome/Layer5-celebration.png)
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/inviBeast/meshery-app-mesh/blob/master/.github/welcome/Meshery_celebration_bg_dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/inviBeast/meshery-app-mesh/blob/master/.github/welcome/Meshery_celebration_bg_light.svg">
```

Comment:
source should be targeting to the original repo, not the Forked, and better to use raw URLs for images

Reply:
ok I will change